### ISSUE TITLE
A second bug located in AdvancedPostProcessing.PostProcessor.create_fieldplot_surface()

### ISSUE THREAD

**Issue by obrienpja** [pyansys/pyaedt#1388]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

I am hitting an exception in AdvancedPostProcessing.PostProcessor.create_fieldplot_surface() that I was not hitting in a previous version (0.4.83). The bug I am seeing occurs on the following line in solutions.py (line 2298 in version 0.5.1):
oname = self._postprocessor.modeler.oeditor.GetObjectNameByFaceID(index)

Here is the exception:
```self._postprocessor.modeler.oeditor.GetObjectNameByFaceID(index)
Traceback (most recent call last):
  File "C:\Anaconda3\envs\pob_qiskit_metal_pyaedt2\lib\site-packages\pyaedt\modules\solutions.py", line 2298, in plotGeomInfo
    oname = self._postprocessor.modeler.oeditor.GetObjectNameByFaceID(index)
  File "<COMObject SetActiveEditor>", line 3, in GetObjectNameByFaceID
pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147024381), None)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<COMObject SetActiveEditor>", line 3, in GetObjectNameByFaceID
pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, None, None, None, 0, -2147024381), None)
```

I am quite sure that this bug was introduced on July 1st in this commit:
https://github.com/pyansys/pyaedt/commit/d94ac54d589a740840c07551b12f0d0e359c8d37
where this line of code was first introduced.

### Steps To Reproduce

1. Instantiate an instance of Hfss() that has an Eigenmode setup
2. Analyze that setup 
3. Then run the post processor's create_fieldplot_surface() method using a syntax like this:
postProcessor.create_fieldplot_surface(objlist=[object_id], quantityName='Mag_E'), where object_id is the id of the object in your design for which you would like to plot the magnitude of the E field.

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.9

### Installed packages

addict @ file:///home/conda/feedstock_root/build_artifacts/addict_1636818143388/work
aiohttp==3.8.1
aiosignal==1.2.0
alabaster==0.7.12
anyio==3.6.1
appdirs==1.4.4
argon2-cffi @ file:///home/conda/feedstock_root/build_artifacts/argon2-cffi_1640817743617/work
argon2-cffi-bindings @ file:///D:/bld/argon2-cffi-bindings_1649500527917/work
asttokens @ file:///home/conda/feedstock_root/build_artifacts/asttokens_1618968359944/work
async-timeout==4.0.2
attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1640799537051/work
Babel @ file:///home/conda/feedstock_root/build_artifacts/babel_1655419414885/work
backcall @ file:///home/conda/feedstock_root/build_artifacts/backcall_1592338393461/work
backports.functools-lru-cache @ file:///home/conda/feedstock_root/build_artifacts/backports.functools_lru_cache_1618230623929/work
beautifulsoup4 @ file:///home/conda/feedstock_root/build_artifacts/beautifulsoup4_1649463573192/work
bleach @ file:///home/conda/feedstock_root/build_artifacts/bleach_1656355450470/work
branca @ file:///home/conda/feedstock_root/build_artifacts/branca_1650311966802/work
brotlipy @ file:///D:/bld/brotlipy_1648854347500/work
cached-property @ file:///home/conda/feedstock_root/build_artifacts/cached_property_1615209429212/work
certifi==2022.6.15
cffi @ file:///D:/bld/cffi_1656782940775/work
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1655906222726/work
click @ file:///D:/bld/click_1651215332191/work
click-plugins==1.1.1
cligj @ file:///home/conda/feedstock_root/build_artifacts/cligj_1633637764473/work
clr-loader==0.1.7
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1655412516417/work
cryptography @ file:///D:/bld/cryptography_1657174142266/work
cycler @ file:///home/conda/feedstock_root/build_artifacts/cycler_1635519461629/work
Cython @ file:///D:/bld/cython_1652809486376/work
debugpy @ file:///D:/bld/debugpy_1649586532851/work
decorator @ file:///home/conda/feedstock_root/build_artifacts/decorator_1641555617451/work
defusedxml @ file:///home/conda/feedstock_root/build_artifacts/defusedxml_1615232257335/work
descartes==1.1.0
docutils @ file:///D:/bld/docutils_1648737820009/work
entrypoints @ file:///home/conda/feedstock_root/build_artifacts/entrypoints_1643888246732/work
executing @ file:///home/conda/feedstock_root/build_artifacts/executing_1646044401614/work
fastjsonschema @ file:///home/conda/feedstock_root/build_artifacts/python-fastjsonschema_1641751198313/work/dist
Fiona==1.8.21
flit_core @ file:///home/conda/feedstock_root/build_artifacts/flit-core_1645629044586/work/source/flit_core
folium @ file:///home/conda/feedstock_root/build_artifacts/folium_1638560701663/work
fonttools @ file:///D:/bld/fonttools_1657249488788/work
frozenlist==1.3.0
GDAL==3.5.1
gdspy @ file:///D:/bld/gdspy_1654258956731/work
geopandas @ file:///home/conda/feedstock_root/build_artifacts/geopandas_1655828570365/work
h5py @ file:///D:/bld/h5py_1656337121838/work
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1642433548627/work
imageio==2.19.3
imagesize @ file:///home/conda/feedstock_root/build_artifacts/imagesize_1656939531508/work
importlib-metadata @ file:///D:/bld/importlib-metadata_1653252994979/work
importlib-resources @ file:///home/conda/feedstock_root/build_artifacts/importlib_resources_1655356668708/work
ipykernel @ file:///D:/bld/ipykernel_1657295151080/work
ipython @ file:///D:/bld/ipython_1653755095589/work
ipython-genutils==0.2.0
ipywidgets @ file:///home/conda/feedstock_root/build_artifacts/ipywidgets_1655973868664/work
jedi @ file:///D:/bld/jedi_1649067336192/work
Jinja2 @ file:///home/conda/feedstock_root/build_artifacts/jinja2_1654302431367/work
joblib @ file:///home/conda/feedstock_root/build_artifacts/joblib_1633637554808/work
json5==0.9.8
jsonschema @ file:///home/conda/feedstock_root/build_artifacts/jsonschema-meta_1657629641118/work
jupyter @ file:///D:/bld/jupyter_1637233334138/work
jupyter-client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1654730843242/work
jupyter-console @ file:///home/conda/feedstock_root/build_artifacts/jupyter_console_1655961255101/work
jupyter-core @ file:///D:/bld/jupyter_core_1652365483950/work
jupyter-server==1.18.1
jupyterlab==3.4.3
jupyterlab-pygments @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_pygments_1649936611996/work
jupyterlab-server==2.15.0
jupyterlab-widgets @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_widgets_1655961217661/work
kiwisolver @ file:///D:/bld/kiwisolver_1655141740116/work
mapclassify @ file:///home/conda/feedstock_root/build_artifacts/mapclassify_1627363180059/work
MarkupSafe @ file:///D:/bld/markupsafe_1648737731599/work
matplotlib @ file:///D:/bld/matplotlib-suite_1651609712471/work
matplotlib-inline @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-inline_1631080358261/work
mistune @ file:///D:/bld/mistune_1635844812333/work
mpmath @ file:///home/conda/feedstock_root/build_artifacts/mpmath_1612895720168/work
multidict==6.0.2
munch==2.5.0
munkres==1.1.4
nbclassic==0.4.3
nbclient @ file:///home/conda/feedstock_root/build_artifacts/nbclient_1656688109017/work
nbconvert @ file:///home/conda/feedstock_root/build_artifacts/nbconvert-meta_1649676641343/work
nbformat @ file:///home/conda/feedstock_root/build_artifacts/nbformat_1651607001005/work
nest-asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1648959695634/work
networkx @ file:///home/conda/feedstock_root/build_artifacts/networkx_1655215393654/work
notebook @ file:///home/conda/feedstock_root/build_artifacts/notebook_1654636967533/work
notebook-shim==0.1.0
numpy @ file:///D:/bld/numpy_1657483944523/work
packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1637239678211/work
pandas @ file:///D:/bld/pandas_1656001224800/work
pandocfilters @ file:///home/conda/feedstock_root/build_artifacts/pandocfilters_1631603243851/work
parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1638334955874/work
patsy @ file:///home/conda/feedstock_root/build_artifacts/patsy_1632667180946/work
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1602536217715/work
Pillow @ file:///D:/bld/pillow_1657007262169/work
Pint @ file:///home/conda/feedstock_root/build_artifacts/pint_1650914444525/work
plumbum==1.7.2
ply==3.11
prometheus-client @ file:///home/conda/feedstock_root/build_artifacts/prometheus_client_1649447152425/work
prompt-toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1656332401605/work
psutil @ file:///D:/bld/psutil_1653089388778/work
pure-eval @ file:///home/conda/feedstock_root/build_artifacts/pure_eval_1642875951954/work
pyaedt==0.5.1
pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1636257122734/work
pyEPR-quantum @ file:///home/conda/feedstock_root/build_artifacts/pyepr-quantum_1653683926245/work
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1650904496387/work
pyOpenSSL @ file:///home/conda/feedstock_root/build_artifacts/pyopenssl_1643496850550/work
pyparsing @ file:///home/conda/feedstock_root/build_artifacts/pyparsing_1652235407899/work
pyproj @ file:///D:/bld/pyproj_1655448289213/work
PyQt5==5.15.7
PyQt5-sip @ file:///D:/bld/pyqt-split_1657234606115/work/pyqt_sip
pyrsistent @ file:///D:/bld/pyrsistent_1649013512562/work
PySide2==5.15.5
PySocks @ file:///D:/bld/pysocks_1648857426124/work
python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/python-dateutil_1626286286081/work
pythonnet==3.0.0rc2
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1647961439546/work
pyvista==0.35.1
pywin32==303
pywinpty @ file:///D:/bld/pywinpty_1657201485954/work/target/wheels/pywinpty-2.0.6-cp310-none-win_amd64.whl
PyYAML @ file:///D:/bld/pyyaml_1648757288117/work
pyzmq @ file:///D:/bld/pyzmq_1656183676913/work
qiskit-metal==0.1.0
qtconsole @ file:///home/conda/feedstock_root/build_artifacts/qtconsole-base_1654465674650/work
QtPy @ file:///home/conda/feedstock_root/build_artifacts/qtpy_1651524593465/work
qutip==4.7.0
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1656534056640/work
rpyc==5.0.1
Rtree @ file:///D:/bld/rtree_1649367817754/work
scikit-learn @ file:///D:/bld/scikit-learn_1652976942320/work
scipy @ file:///C:/bld/scipy_1653074056978/work
scooby==0.5.12
scqubits @ file:///D:/bld/scqubits_1656753946679/work
seaborn @ file:///home/conda/feedstock_root/build_artifacts/seaborn-split_1629095986539/work
Send2Trash @ file:///home/conda/feedstock_root/build_artifacts/send2trash_1628511208346/work
Shapely==1.8.2
shiboken2==5.15.5
sip @ file:///D:/bld/sip_1655765040828/work
six @ file:///home/conda/feedstock_root/build_artifacts/six_1620240208055/work
sniffio==1.2.0
snowballstemmer @ file:///home/conda/feedstock_root/build_artifacts/snowballstemmer_1637143057757/work
soupsieve @ file:///home/conda/feedstock_root/build_artifacts/soupsieve_1638550740809/work
Sphinx @ file:///home/conda/feedstock_root/build_artifacts/sphinx_1655452194084/work
sphinx-rtd-theme @ file:///home/conda/feedstock_root/build_artifacts/sphinx_rtd_theme_1617698677674/work
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp @ file:///home/conda/feedstock_root/build_artifacts/sphinxcontrib-htmlhelp_1621704829796/work
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml @ file:///home/conda/feedstock_root/build_artifacts/sphinxcontrib-serializinghtml_1649380998999/work
stack-data @ file:///home/conda/feedstock_root/build_artifacts/stack_data_1655315839047/work
statsmodels @ file:///D:/bld/statsmodels_1644535944524/work
sympy @ file:///D:/bld/sympy_1647802833311/work
terminado @ file:///D:/bld/terminado_1652790796353/work
threadpoolctl @ file:///home/conda/feedstock_root/build_artifacts/threadpoolctl_1643647933166/work
tinycss2 @ file:///home/conda/feedstock_root/build_artifacts/tinycss2_1637612658783/work
toml @ file:///home/conda/feedstock_root/build_artifacts/toml_1604308577558/work
tornado @ file:///D:/bld/tornado_1656937966227/work
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1649051611147/work
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1655411388954/work
typing_extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1656706066251/work
unicodedata2 @ file:///D:/bld/unicodedata2_1649112137711/work
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1657224465922/work
vtk==9.2.0rc1
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1600965781394/work
webencodings==0.5.1
websocket-client==1.3.3
widgetsnbextension @ file:///home/conda/feedstock_root/build_artifacts/widgetsnbextension_1655939017940/work
win-inet-pton @ file:///D:/bld/win_inet_pton_1648771897787/work
wslink==1.6.6
xarray @ file:///home/conda/feedstock_root/build_artifacts/xarray_1646254587173/work
xyzservices @ file:///home/conda/feedstock_root/build_artifacts/xyzservices_1655861233352/work
yarl==1.7.2
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1649012893348/work

**Comment by maxcapodi78** [comment]
@obrienpja you are right. I've introduced a regression by adding the possibility to port surface fields on on models. 
I've fixed in #1423

**Comment by obrienpja** [comment]
Excellent, thank you for the fix!
